com.unity.netcode.gameobjects/Runtime/SceneManagement/SceneEventData.cs

```diff
@@ -475,7 +475,8 @@ internal void Deserialize(FastBufferReader reader)
                             // We store off the trailing in-scene placed serialized NetworkObject data to
                             // be processed once we are done loading.
                             m_HasInternalBuffer = true;
-                            InternalBuffer = new FastBufferReader(reader.GetUnsafePtrAtCurrentPosition(), Allocator.TempJob, reader.Length - reader.Position);
+                            // We use Allocator.Persistent since scene loading could take longer than 4 frames
+                            InternalBuffer = new FastBufferReader(reader.GetUnsafePtrAtCurrentPosition(), Allocator.Persistent, reader.Length - reader.Position);
```

Comment:
And I'm assuming we want and need it to live that extended period?

Reply:
Yeah.   Allocator.TempJob will start warning if you hold onto that for longer than 4 frames.  That buffer is to be used for scene loading which could take dozens to hundreds of frames depending upon the size of the scene being loaded.